tests/es/homeassistant_HassTurnOn.yaml

```diff
@@ -7,3 +7,15 @@ tests:
       name: "HassTurnOn"
       slots:
         name: "fan.ventilador_techo"
+  - sentences:
+      - "abre la cortina izquierda"
```

Comment:
Cover specific sentences should be added to `cover_HassTurnOn.yaml`

Reply:
Removed sentence.